pipelines/rj_smas__disparo_pic_lembrete/schemas.py

```
@@ -0,0 +1,165 @@
+
+        Args:
+            v: String do nome da campanha
+
+        Returns:
+            String do nome da campanha validada
+
+        Raises:
+            ValueError: Se for vazio ou apenas espaços
+        """
+        if not v.strip():
+            raise ValueError("Nome da campanha não pode ser vazio ou apenas espaços")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)